aten/src/ATen/templates/BackendSelectRegister.cpp

```diff
@@ -0,0 +1,22 @@
+// This is used for factory methods dispatching. We register ops with a high priority dispatch key,
+// which makes the dispatcher to dispatch here. We compute the dispatch key 'manually' and navigate to the
+// correct backend.
```

Comment:
this description isn't clear at all.  There's no reason to be so general, you can give the specific examples.

For example:
"We register ops with a higher priority dispatch key (BackendSelect) than the usual backend-specific keys (e.g. CPUTensorId) which makes calls to the factory functions dispatch to here.  We then 'manually' compute a lower-priority to re-dispatch to (e.g. CPUTensorId) to get to the eventually correct backend."